### PR TITLE
Migrate APIMANAGER3706ApplicationUpdateTestCase

### PR DESCRIPTION
**Pull request by vithu30**

### CODE REVIEW

**Comment by mushthaq33** [comment]
We won't be needing this test case as most  of this scenarios are covered from other test cases. Can we remove this test case?

**Comment by vithu30** [comment]
Yes, application update is covered in the testcase - **GrantTypeTokenGenerateTestCase**. But we need the test to check the consumer secret re-generation REST API. Shall we only have RegenerateConsumerSecret testcase?

**Comment by mushthaq33** [comment]
Yes +1 for that.

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/application/ApplicationConsumerSecretRegenerateTestCase.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        Assert.assertEquals(regenerateResponse.getStatusCode(), Response.Status.OK.getStatusCode(),
+                "Error when re-generating consumer secret for application: " + applicationID);
+        Assert.assertNotNull(regenerateResponse.getData().getConsumerSecret());
+        Assert.assertNotEquals(consumerSecret, regenerateResponse.getData().getConsumerSecret());
+    }
+
+    @AfterClass(alwaysRun = true)
+    public void destroy() throws Exception {
+        restAPIStore.deleteApplication(applicationID);
+        super.cleanUp();
```
Comment (mushthaq33) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/application/ApplicationConsumerSecretRegenerateTestCase.java]: Lets use super.cleanUpUsingRest() here

Comment (vithu30) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/application/ApplicationConsumerSecretRegenerateTestCase.java]: Fixed in  a08ae69

**Pull request merged**